RPIPs/RPIP-46.md

```diff
@@ -29,91 +29,121 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
 3. `reth_share` SHALL be defined as `100% - reth_commission`
 4. Distributions of revenue from borrowed ETH MUST respect the defined shares
    1. If shares change between claims, distributions MUST make an effort to account for the different values. For example, a distribution could use a duration-weighted average share. Approximations MAY be used where they significantly reduce complexity and/or costs.
    2. Legacy minipools are an exception and SHALL continue to support earlier distribution methodologies 
-5. `node_operator_commission_share`, `node_operator_commission_share_council_adder`, and `surplus_share`, SHALL be updateable by an address in the `allowlisted_controllers` array
+5. `node_operator_commission_share`, `node_operator_commission_share_council_adder`, and `voter_share`, SHALL be updateable by an address in the `allowlisted_controllers` array
    1. This functionality SHALL not be used without a separate pDAO vote to enable a controller and add it to the list
 6. The `node_operator_commission_share_council_adder` setting SHALL only allow values where:
    1. 0% ≤ `node_operator_commission_share_council_adder` ≤ `max_node_operator_commission_share_council_adder` 
-   2. `node_operator_commission_share_council_adder` ≤ `surplus_share`
+   2. `node_operator_commission_share_council_adder` ≤ `voter_share`
 7. The `node_operator_commission_share_council_adder` setting SHALL be controllable by the security council without requiring a delay
 8. The security council SHOULD increment `node_operator_commission_share_council_adder` by 0.5% if the deposit pool is over half-full for the majority of a 2-week period with a constant `node_operator_commission_share + node_operator_commission_share_council_adder`
     1. The security council SHALL NOT otherwise change `node_operator_commission_share_council_adder`
-9. `voter_share` SHALL NOT be controllable by pDAO vote and SHALL be initialized to 5%
-10. The initial pDAO settings SHALL be:
-    1. `node_operator_commission_share`: 3.5%
+9. The initial pDAO settings SHALL be:
+    1. `node_operator_commission_share`: 5%
     2. `node_operator_commission_share_council_adder`: 0%
-    3. `surplus_share`: 5.5%
+    3. `voter_share`: 9%
     4. `max_node_operator_commission_share_council_adder`: 1%
     5. `allowlisted_controllers`: []
 
-### Inflation
+### RPL rewards
+1. The RPL reward system specified in RPIP-30 SHALL continue
+   1. All staked RPL SHALL be counted for this purpose (both "legacy staked RPL" and "megapool staked RPL" as defined in [RPIP-43](RPIP-43.md))
+
+### Revenue share vote
+Prior to the release of Saturn 1, a ranked-choice vote MUST be held to select a mechanism for revenue share to RPL
+1. The choices SHALL include retaining the state from Saturn 1
+2. The choices MAY include [RPIP-45: RPL Burn](RPIP-45.md), [RPIP-50: RPL LP](RPIP-50.md), and novel options created for this vote
```

Comment:
```suggestion
2. The choices SHOULD include [RPIP-45: RPL Burn](RPIP-45.md), [RPIP-50: RPL LP](RPIP-50.md)
3. The choices MAY include as-yet unwritten options in RPIP form. 
```
I'd argue the first should be SHOULD over MAY, we've talked about Burn and LP enough that I think people are expecting to vote on them unless there is a concrete reason we can explain not to, which imo is closer to SHOULD than MAY.

Reply:
Maybe rewrite without the as-yet.